README.md

```diff
@@ -46,25 +46,13 @@ __Don't change anything, just add KubeStellar!__
 - __Remain compatible__ with cloud-native solutions
 
 ## KubeStellar virtual clusters (Spaces) are our secret
+TODO: Need to update!
 - KubeStellar uses lightweight virtual clusters (Spaces) that run inside the KubeStellar hosting cluster
 - Standard Kubernetes clusters have __2-300__ api-resources, KubeStellar Spaces have only __40__
 - Fewer api-resources mean resources remain inactive (denatured) – they do not expand into other resources like replicasets, pods, etc.
 - Denaturing is the key to accepting native, unbundled Kubernetes resources as input without running them
 - Unbundled resources are the default and preferred output of most cloud-native tools making KubeStellar use and integration easy
 
-## QuickStart
-
```

Comment:
Maybe leave the section and put TBD underneath ?

Reply:
done